config/resource_suppressions.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+
+<!DOCTYPE suppressions PUBLIC
+    "-//Puppy Crawl//DTD Suppressions 1.1//EN"
+    "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
+
+<suppressions>
+    
+      <!-- Suppressions inside test and it folders -->
```

Comment:
This whole configuration is for test and it folders. So comment is redundant.
Make comment specify it is for why we can't add newline to end of files or none at all.